src/Responder/FormattedResponder.php

```diff
@@ -49,8 +49,7 @@ public function __invoke(
         PayloadInterface $payload
     ) {
         if ($this->hasOutput($payload)) {
-            $formatter = $this->formatter($request);
-            $response = $this->format($response, $formatter, $payload);
```

Comment:
@shadowhand I agree with you, I will. Just too many changes intersect. 

I would like to hear something about the changes.. (is it worth it?)